README.md

```diff
@@ -256,10 +256,16 @@ $ sudo make test
 It is highly recommended to run these tests either as part of a CI/CD system e.g. circleci or on
 a immutable infrastructure e.g vagrant VMs.
 
+Instead of running all tests, you can also running specified test only by (replace the test name)
+
+```
+$ sudo make test tests/001-test-redis.sh
```

Comment:
I am seeing some issues when running an individual test using `sudo make test`.
e.g. when I run:

```
vagrant@vagrant:~/go/src/github.com/Roblox/nomad-driver-containerd$ sudo make test tests/003-test-capabilities.sh
```
This runs all the tests (and **not** just the individual test), and in the end prints the following message:

```
make: Nothing to be done for 'tests/003-test-capabilities.sh'.
```

My host OS is Mac (Not sure if that is a cause for concern).
Is this working for you?
**NOTE**: This is the same behavior on the `master` branch, so nothing to do with this PR/branch.

Other option is, we can just document running individual test without `Makefile` command.

```
You can also run an individual test by specifying the test name. e.g.

$ cd tests
$ ./run_tests.sh 001-test-redis.sh
```

Reply:
Oh, sorry, i actually running is `sudo tests/run_tests.sh test/001-test-redis.sh` for it can complete the test name via command line.

What's the different between the two commands? just curious.

Reply:
Nothing really, I was just changing directory (`cd`) to the `target` directory (`tests`) first, so it's easier for the user to read the instructions.
In the end all it is doing is [`reading the individual test name`](https://github.com/Roblox/nomad-driver-containerd/blob/master/tests/run_tests.sh#L53) and passing it to [`run_test`](https://github.com/Roblox/nomad-driver-containerd/blob/master/tests/run_tests.sh#L58)

In that case, can we update the `README.md` with the text I suggested?

Reply:
Yes, sure.